frontend/src/lib/constants.tsx

```diff
@@ -22,12 +22,7 @@ export const NON_VALUES_ON_SERIES_DISPLAY_TYPES = [
 ]
 
 /** Display types for which a percent stack view is available. */
-export const PERCENT_STACK_VIEW_DISPLAY_TYPE = [
-    ChartDisplayType.ActionsBar,
-    ChartDisplayType.ActionsLineGraph,
-    ChartDisplayType.ActionsAreaGraph,
```

Comment:
Why remove the option from line graphs and more importantly area graphs?

Reply:
This option require a stacked view, which doesn't work with line graphs. But it does actually work for area graphs, so brought these back. Thanks for pointing out!